release/release_gold.sh

```diff
@@ -132,6 +132,28 @@ function tag_release {
 
 		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
 	fi
+
+	if (! curl \
+			"https://api.github.com/repos/liferay/${repository}/git/refs" \
+			--data-raw '
+				{
+					"message": "",
+					"ref": "refs/tags/'"${LIFERAY_RELEASE_VERSION}"'",
+					"sha": "'$(lc_get_property "${_PROJECTS_DIR}"/liferay-portal-ee/release.properties release.tool.sha)'"
```

Comment:
@lucasmiranda0, we should still get the property `git.hash.liferay-portal-ee` which is available on the web, like in https://releases.liferay.com/dxp/2024.q1.3/release.properties. Check the code below for a more detailed input:
```
release_properties_file=$(lc_download "https://releases.liferay.com/${LIFERAY_RELEASE_PRODUCT_NAME}/${_PRODUCT_VERSION}/release.properties")

local git.hash.liferay-portal-ee=$(lc_get_property "${release_properties_file}" git.hash.liferay-portal-ee)
```

Please make sure this code is properly handling potential issues, please check [here](https://github.com/liferay/liferay-docker/blob/d17272c5a8a8bc9687172d924494f4d7c40349f3/release/_releases_json.sh#L71-L88) to get some inspiration.